CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Changes
 
+### 1.3.7
```

Comment:
Please call it `Unreleased` for now. It will at least be `1.4.0` as this is a feature, but let's see what we'll get in at this version.

Reply:
Sure thing. Will you be releasing this as a preview, though? I want to be able to use this feature 😄